cli/command/environment.go

```diff
@@ -63,6 +63,16 @@ func (e *EnvironmentCommand) Command() cli.Command {
 				Usage:     "delete an environment",
 				ArgsUsage: "ENVIRONMENT_NAME",
 				Action:    e.delete,
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "r",
```

Comment:
This is how you specify aliases with the cli package
```go
cli.BoolFlag{
    Name: "r, recursive",
    ...
}
```

Reply:
✅